siddpubs-journal.bib

```diff
@@ -17,6 +19,26 @@ @string{auro
 @string{ral  = "{IEEE} Robotics and Automation Letters"}
 @string{tro  = "{IEEE} Transactions on Robotics"}
 @string{thri = "{ACM} Transactions on Human-Robot Interaction"}
+@string{jmlr = "Journal of Machine Learning Research"}
+
+
+
+@article{roulet2025ilqriddp,
+  title={On Global and Local Convergence of Iterative Linear Quadratic Optimization Algorithms for Discrete Time Nonlinear Control}, 
+  author={Vincent Roulet and Srinivasa, S.S. and Maryam Fazel and Zaid Harchaoui},
+  journal= jmlr,
+  year={2025}
+}
+
+@article{roulet2024ilqrtemplates,
+     author = {Vincent Roulet and Srinivasa, S.S> and Maryam Fazel and Zaid Harchaoui},
```

Comment:
Same question about full name vs abbreviation. Also, is the `>` here needed, or is it a typo?
```suggestion
     author = {Vincent Roulet and Srinivasa, S.S and Maryam Fazel and Zaid Harchaoui},
```